File: fix_lineendings.py
import os

file_path = 'js/app.js'
with open(file_path, 'rb') as f:
    content = f.read()

bad_count = content.count(b'\r\r\n')
print(f'Righe con line ending errato (\\r\\r\\n): {bad_count}')

if bad_count > 0:
    fixed_content = content.replace(b'\r\r\n', b'\r\n')
    with open(file_path, 'wb') as f:
        f.write(fixed_content)
    print('Corretti con successo.')
else:
    print('Nessun problema trovato.')

File: fix_lineendings.js
const fs = require('fs');
const file = 'js/app.js';
const c = fs.readFileSync(file);
const str = c.toString('binary');
const matches = str.match(/\r\r\n/g);
const bad = matches ? matches.length : 0;
console.log('Righe errate:', bad);
if (bad > 0) {
    const fixed = str.replace(/\r\r\n/g, '\r\n');
    fs.writeFileSync(file, Buffer.from(fixed, 'binary'));
    console.log('Corrette.');
} else {
    console.log('OK, nessun problema.');
}
